templates/js/copilot-plugin-from-scratch-api-key/keyGen.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# The file path where the API Key is located.
+file_path="env/.env.$1.user"
```

Comment:
Yes, we specify using Bash to handle the script in the script action, so it can run across different platforms.

```
  # Set required variables for local launch
  - uses: script
    with:
      shell: bash
      run:
        ./keyGen.sh ${{TEAMSFX_ENV}};
```